examples/jsm/loaders/GLTFLoader.js

```diff
@@ -2205,6 +2205,7 @@ var GLTFLoader = ( function () {
 				pointsMaterial.color.copy( material.color );
 				pointsMaterial.map = material.map;
 				pointsMaterial.lights = false; // PointsMaterial doesn't support lights yet
+				pointsMaterial.sizeAttenuation = false; // glTF spec says points should be 1px
```

Comment:
That change looks suspicious. A rebase should solve this issue.

Reply:
Wait! It's just the update `jsm` version of `GLTFLoader`. I think it's best if you undo this change for clarity. If you execute `node utils/modularize.js` you might update JSM files that do not belong to your PR.

Reply:
So should I undo it manually?
And is there a way to run modularize.js on a single file? Is that what I should have done?

Reply:
Only include files that are relevant to your change in commits/PRs.

But in this case it's okay, that file was pending an update anyway. Will merge now.

Reply:
Ok, I'll pay better attention in the future